9.1.индексация/9.1.15.py

```diff
@@ -0,0 +1,6 @@
+num = int(input())  
+double = ""            
+while num > 0:
+    double = str(num % 2) + double   
```

Comment:
Не забываем о расширенных операторах присваивания :)
```suggestion
    double += str(num % 2)
```